app/screens/home/search/results/results.tsx

```diff
@@ -2,20 +2,35 @@
 // See LICENSE.txt for license information.
 
 import React, {useCallback, useMemo} from 'react';
-import {FlatList, ListRenderItemInfo, NativeScrollEvent, NativeSyntheticEvent, Text, View} from 'react-native';
-import Animated from 'react-native-reanimated';
+import {StyleSheet, FlatList, ListRenderItemInfo, NativeScrollEvent, NativeSyntheticEvent, Text, StyleProp, View, ViewStyle} from 'react-native';
+import Animated, {useDerivedValue} from 'react-native-reanimated';
 
+import File from '@components/files/file';
 import NoResultsWithTerm from '@components/no_results_with_term';
 import DateSeparator from '@components/post_list/date_separator';
 import PostWithChannelInfo from '@components/post_with_channel_info';
 import {Screens} from '@constants';
 import {useTheme} from '@context/theme';
-import {PostModel} from '@database/models/server';
+import {PostModel, ChannelModel} from '@database/models/server';
+import {useIsTablet} from '@hooks/device';
+import {useImageAttachments} from '@hooks/files';
+import {isImage, isVideo} from '@utils/file';
+import {fileToGalleryItem, openGalleryAtIndex} from '@utils/gallery';
+import {getViewPortWidth} from '@utils/images';
 import {getDateForDateLine, isDateLine, selectOrderedPosts} from '@utils/post_list';
+import {TabTypes, TabType} from '@utils/search';
+import {preventDoubleTap} from '@utils/tap';
 
-import FileCard from './fileCard';
 import Loader from './loader';
 
+const styles = StyleSheet.create({
+    flex: {flex: 1},
+    container: {
+        flex: 1,
+        marginHorizontal: 20,
+    },
+});
+
 const notImplementedComponent = (
```

Comment:
to be removed

Reply:
https://mattermost.atlassian.net/browse/MM-45614